cdap-docs/faq/source/general.rst

```diff
@@ -66,52 +60,43 @@ advantages of using CDAP include:
    lifecycle and scale of your application.
 
 
-Platforms and Language
-======================
-
-.. rubric:: What Platforms are Supported by the Cask Data Application Platform SDK?
-
-The CDAP SDK can be run on Mac OS X, Linux or Windows platforms.
 
-.. rubric:: What programming languages are supported by CDAP?
+What platforms are supported by the Cask Data Application Platform SDK?
+-----------------------------------------------------------------------
+The CDAP SDK can be run on Mac OS X, Linux, or Windows platforms.
 
+What programming languages are supported by CDAP?
+-------------------------------------------------
 CDAP currently supports Java for developing applications.
 
-.. rubric:: What Version of Java SDK is Required by CDAP?
-
+What version of Java SDK is required by CDAP?
+---------------------------------------------
 The latest version of the JDK or JRE version 7 or version 8 must be installed
 in your environment; we recommend the Oracle JDK.
 
-.. rubric:: What Version of Node.JS is Required by CDAP?
-
-The version of Node.js must be from |node-js-version|; we recommend |recommended-node-js-version|.
+What version of Node.js is required by CDAP?
+--------------------------------------------
+The version of `Node.js <https://nodejs.org/>`__ must be from |node-js-version|; we recommend |recommended-node-js-version|.
 
 
-Hadoop
-======
-
-.. rubric:: I have a Hadoop cluster in my data center, can I run CDAP that uses my Hadoop cluster?
-
+I have a Hadoop cluster in my data center, can I run CDAP that uses my Hadoop cluster?
+--------------------------------------------------------------------------------------
 Yes. You can install CDAP on your Hadoop cluster. See :ref:`install`.
 
-.. rubric:: What Hadoop distributions can CDAP run on?
-
+What Hadoop distributions can CDAP run on?
+------------------------------------------
 CDAP |version| has been tested on and supports CDH 5.0.0 through 5.4.x; HDP 2.0 through 2.3; 
 MapR 4.1 and 5.0, and Apache Bigtop 0.8.0. 
```

Comment:
This is no longer accurate. CDAP 3.1 dropped support for HBase 0.94, which is what is shipped in BigTop 0.8.0... however, we should work on BigTop 1.0, instead.